src/keymap.cpp

```diff
@@ -482,13 +482,17 @@ KeyMap::KeyMap(unsigned flags)
 	 * list above.
 	 */
 	LOG(Level::DEBUG, "KeyMap::KeyMap: flags = %x", flags);
-	for (unsigned int j = 1; contexts[j] != nullptr; j++) {
-		std::string ctx(contexts[j]);
-		for (int i = 0; opdescs[i].op != OP_NIL; ++i) {
-			if (opdescs[i].flags &
-				(flags | KM_INTERNAL | KM_SYSKEYS)) {
-				keymap_[ctx][opdescs[i].default_key] =
-					opdescs[i].op;
+	for (int i = 0; opdescs[i].op != OP_NIL; ++i) {
+		OpDesc op_desc = opdescs[i];
```

Comment:
`op_desc`, `context`, and `context_flag` can all be marked `const`.

Reply:
Done, thanks